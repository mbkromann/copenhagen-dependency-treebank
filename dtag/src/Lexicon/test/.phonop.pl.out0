fakler = fakler = [fakkel, -~e, er]
fandtes = fandtes = [fandt, ([^aeiouyæøå])>$1e, s]
huskedes = huskedes = [huskede, ([^aeiouyæøå])>$1e, s]
laver = laver = [lav, er]
laves = laves = [lav, er, r>s]
lytter = lytter = [lyt, ~, er]
mand = mand = [mand, , ^æ]
mænd = mænd = [mand, ^æ]
mændene = mændene = [mand, ^æ, , ene]
sprukket = sprukket = [spræk, ^u, ~, et]
sås = sås = [så, ([^aeiouyæøå])>$1e, s]
talte = talte = [tæl, ^a, te]
tidsler = tidsler = [tidsel, -e, er]
