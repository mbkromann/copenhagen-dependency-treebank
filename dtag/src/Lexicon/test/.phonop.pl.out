ERROR! cannot read file /opt/dta/arcs.header in Graph->readfile
check that DTAGHOME is set correctly!
ERROR! cannot read file /opt/dta/arcs.trailer in Graph->readfile
check that DTAGHOME is set correctly!
fakler = fakler = [fakkel, -~e, er]
fandtes = fandtes = [fandt, ([^aeiouyæøå])>$1e, s]
huskedes = huskedes = [huskede, ([^aeiouyæøå])>$1e, s]
laver = laver = [lav, er]
laves = laves = [lav, er, r>s]
lytter = lytter = [lyt, ~, er]
mand = mand = [mand, , ^æ]
mænd = mænd = [mand, ^æ]
mændene = mændene = [mand, ^æ, , ene]
sprukket = sprukket = [spræk, ^u, ~, et]
sås = sås = [så, ([^aeiouyæøå])>$1e, s]
talte = talte = [tæl, ^a, te]
tidsler = tidsler = [tidsel, -e, er]
